selfdrive/car/hyundai/values.py

```diff
@@ -653,12 +653,6 @@ def match_fw_to_car_fuzzy(live_fw_versions, vin, offline_fw_versions) -> set[str
 FW_QUERY_CONFIG = FwQueryConfig(
   requests=[
     # TODO: add back whitelists
-    # CAN queries (OBD-II port)
-    Request(
-      [HYUNDAI_VERSION_REQUEST_LONG],
-      [HYUNDAI_VERSION_RESPONSE],
-    ),
-
     # CAN & CAN-FD queries (from camera)
     Request(
```

Comment:
send different query/tester present and see if the OBD HYUNDAI_VERSION_REQUEST_LONG response still comes back (failure to multiplex?)

Reply:
there's still an ECU on the bus that occasionally sends random queries to the camera we'll have to work around